lib/assets/javascripts/cable.coffee.erb

```diff
@@ -4,5 +4,8 @@
 @Cable =
   INTERNAL: <%= ActionCable::INTERNAL.to_json %>
 
-  createConsumer: (url) ->
+  createConsumer: (url = @defaultUrl()) ->
     new Cable.Consumer url
+
+  defaultUrl: ->
+    document.querySelector("meta[name='action-cable-url']")?.getAttribute("content")
```

Comment:
Naming here suggests there's a default URL, but really it needs to be configured.

How about:
```coffee
createConsumer: (url = @getConfig("url")) ->
  ...
  
getConfig: (name) ->
  element = document.head.querySelector("meta[name='action-cable-#{name}']")
  element?.getAttribute("content")
```